Styles/styles.css

```diff
@@ -79,15 +107,306 @@ header > .logo {
 .social-media-icons > li {
   width: 100px;
   margin-bottom: 3.25rem;
+  margin-top: -3rem;
 }
 
 .social-media-icons > li,
 a {
   list-style: none;
 }
 
-.illustration {
-  position: absolute;
-  top: 38%;
-  left: 65%;
+.recent-works {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 40px;
+  font-weight: 400;
+  text-align: center;
+  padding: 0 0 2.5rem 0;
+  margin-bottom: 3rem;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  border-bottom: 1px solid black;
+}
+
+.work {
+  width: 100%;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 4rem;
+}
+
+.work-title {
+  margin: 0.6rem;
+  padding-bottom: 1rem;
+  text-align: center;
+  font-size: 40px;
+  border-bottom: 1px solid black;
+}
+
+.grid {
+  display: grid;
+
+  /* grid-template-columns: 1fr; */
+  padding: 1.5rem;
+  gap: 30px;
+}
+
+.collection {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.work1 {
+  margin-bottom: 6rem;
+  height: 460px;
+  display: flex;
+  flex-direction: column;
+  row-gap: 10px;
+}
+
+.work1 > h5 {
+  font-family: 'Crete Round', sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+}
+
+.work1 > p {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+}
+
+.work1 > .lists1 {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+
+  /* padding: 1rem; */
+  justify-content: flex-start;
+  column-gap: 10px;
+}
+
+.lists1 > li {
+  border: 1px solid black;
+  color: black;
+  padding: 0.8rem;
+  font-family: 'Poppins', sans-serif;
+  font-size: 12px;
+}
+
+.work1 > button {
```

Comment:
Adding button interactions to your project is part of the requirements for this project, however, another plus is that it can improve the UX :bust_in_silhouette: :art:  and make your project shine :sparkles: 

Please make sure to add the button interactions specified in the template to your project:
![image](https://user-images.githubusercontent.com/92395484/192076563-3cf569ff-3323-4873-9227-2def2b719754.png)